.github/workflows/next-rc.yml

```diff
@@ -0,0 +1,92 @@
+name: Prepare next release candidate
+
+on:
+  push:
+    branches:
+      - release/*
+env:
+  GIT_USER: github-actions[bot]
+  GIT_EMAIL: 41898282+github-actions[bot]@users.noreply.github.com
+  HELM_DOCS_VERSION: 1.11.0
+
+concurrency:
```

Comment:
nice, maybe we can also use that for our e2e tests that cannot be currently executed at the same time.